proposal/zivzone/proposal.rst

```diff
@@ -0,0 +1,131 @@
+===========================
+Proposal for CrudeNN
+===========================
+
+This is a template for a proposal that specifies what your project is and the
```

Comment:
A proposal should not have the placeholder texts left.

1 point needs to be deducted.